src/components/Results/index.js

```diff
@@ -12,10 +12,20 @@ var paginationLabels = {
 class Results extends React.Component {
   render() {
     var results = this.props.results;
-    var renderedResults = map( results.hits, function( r ) {
-      return <Hogan data={ r } key={ r.objectID } template={ this.props.hitTemplate } />;
-    }, this );
-    return <div className="search_list search_results_container row">{ renderedResults }</div>
+    return  ( results && results.hits && results.hits.length > 0 ) ?
```

Comment:
This ternary seems a bit long. I would have go with something like

```javascript
if (results && results.hits && results.hits.length > 0) {
  return this.renderWithResults(results, this.props.hitTemplate);
}
return this.renderNoResults(results, this.props.hitTemplate);
```

(Note that I've also set the same signature to the two methods `(results, template)`.